components/brave_new_tab_ui/containers/newTab/gridTile.tsx

```diff
@@ -54,45 +49,16 @@ class TopSite extends React.PureComponent<Props, {}> {
         title={siteData.title}
         tabIndex={0}
         style={{
-          // Visually inform users that dragging a pinned site is not allowed.
-          cursor: isGridSitePinned(siteData) ? 'not-allowed' : 'grab'
+          // Visually inform users that dragging a site is not allowed.
+          cursor: customLinksEnabled() ? 'grab' : 'not-allowed'
         }}
       >
         <TileActionsContainer>
-          <TileAction onClick={this.onTogglePinnedTopSite.bind(this, siteData)}>
-            {isGridSitePinned(siteData) ? <PinIcon /> : <PinOIcon />}
+          <TileAction onClick={this.onIgnoredTopSite.bind(this, siteData)}>
+            <CloseStrokeIcon />
```

Comment:
Is this the icon we need to re-position?

Reply:
Yup! This is it 😄 It's moved all the way to the left (since I removed icon for pinning) and the container its in needs to be removed... but doing that causes weird behavior